kinto_wizard/validate.py

```diff
@@ -42,7 +42,13 @@ def validate_schema(data, schema, ignore_fields=[]):
 
 def validate_export(config):
     everything_is_fine = True
-    buckets = config.get("buckets", {})
+    if 'buckets' in config:
+        buckets = config.get("buckets", {})
+    else:  # pragma: no cover
+        # Legacy for file before kinto-wizard 4.0
+        logger.warning("Your file seems to be in legacy format. "
+                       "Please add a `buckets:` root namespace.")
```

Comment:
Is it a *root namespace* ?

Level? Object?

Reply:
You called it root level in #59 I can keep that name yes.